packages/server/graphql/types/TeamMember.ts

```diff
@@ -39,7 +39,11 @@ const TeamMember = new GraphQLObjectType<any, GQLContext>({
       type: GraphQLBoolean,
       description: 'true if the user is a part of the team, false if they no longer are'
     },
-    isLead: {type: GraphQLBoolean, description: 'Is user a team lead?'},
+    isLead: {
```

Comment:
We have `isLead` on both the `TeamMember` and the `Team`. Perhaps we should change the name on `Team` to `isViewerLead` as it's always referring to the viewer?